### PR TITLE
core-1177 fix incorrect date formatting in app details.

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: src/components/apps/details/DetailsPanel.js [hunk 1]
```diff
@@ -105,7 +105,11 @@ function DetailsPanel(props) {
                             </GridLabelValue>
                         )}
                         <GridLabelValue label={t("publishedOn")}>
-                            {formatDate(details.integration_date)}
+                            {details.integration_date
+                                ? formatDateObject(
+                                      new Date(details.integration_date)
+                                  )
+                                : "-"}
```
Comment (psarando) [inline src/components/apps/details/DetailsPanel.js:112]: `formatDateObject` already has `-` as a constant for returning an empty date string.
```suggestion
                            {formatDateObject(
                                details.integration_date && new Date(details.integration_date)
                            )}
```

File: src/components/apps/details/DetailsPanel.js [hunk 2]
```diff
@@ -117,7 +121,13 @@ function DetailsPanel(props) {
...
                         </GridLabelValue>
                         <GridLabelValue label={t("detailsLastCompleted")}>
-                            {formatDate(details.job_stats?.job_last_completed)}
+                            {details.job_stats?.job_last_completed
+                                ? formatDateObject(
+                                      new Date(
+                                          details.job_stats.job_last_completed
+                                      )
+                                  )
+                                : "-"}
```
Comment (psarando) [inline src/components/apps/details/DetailsPanel.js:130]:
```suggestion
                            {formatDateObject(
                                details.job_stats?.job_last_completed &&
                                      new Date(
                                          details.job_stats.job_last_completed
                                      )
                                  )}
```

**Review by psarando — APPROVED** [review]
LGTM.
I would just suggest allowing `formatDateObject` to return its own empty date string.

**Comment by sri2k1us** [comment]
Thanks for the review.

**Pull request merged**